modin/experimental/core/execution/dask/implementations/pandas_on_dask/io/io.py

```diff
@@ -123,4 +119,7 @@ def func(df, **kw):
         result = qc._modin_frame.broadcast_apply_full_axis(
             1, func, other=None, new_index=[], new_columns=[], enumerate_partitions=True
         )
+        # pending completion
         result.to_pandas()
+
+    del __make_read  # to not pollute class namespace
```

Comment:
I would suggest `del`-ing it right after last use (after `read_sql = __make_read(...)`), it's much easier for the reader to follow then

Reply:
done